### PR TITLE
Adder WS 3 (addw3)

### PR DESCRIPTION
**Pull request by jacobgkau**
WIP

### CODE REVIEW

**Review by ahoneybun — APPROVED** [review]
LGTM other then the comment about the DC-in port.

**Review by leviport — CHANGES_REQUESTED** [review]

File: src/models/addw3/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Memory
+    - Up to 64GB (2x32GB) dual-channel DDR5 SO-DIMMs @ 4800 MHz
+- Networking
+    - Gigabit Ethernet
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Intel Wi-Fi 6E AX210/AX211
+            - Bluetooth 5.3
+- Power
+    - 280W (20V, 14A) DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (inner)
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/addw3/README.md]: You're right. I changed it to the wording we currently have for bonw14 and serw13. 4dc6a8f

File: src/models/addw3/external-overview.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Keyboard & touchpad overview:
+
+![Keyboard & Touchpad](./img/keyboard-touchpad.webp)
+
+## Materials:
+
+|Part                              |Material |Part Number |
+|----------------------------------|---------|------------|
+|LCD back cover (lid)              |Aluminum |• 15": 6-39-NP501-024<br/>• 17": 6-39-PD701-023<br/>or 6-39-PD711-021|
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/addw3/external-overview.md]: 4dc6a8f

File: src/models/addw3/external-overview.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+|LCD front cover (bezel)           |Plastic  |• 15": 6-39-NP501-033<br/>• 17": 6-39-PD701-012|
+|Top case (palm rests/port covers) |Plastic  |• 15": 6-78-NP50SNE2-010<br/>• 17": 6-78-PD70SNEG2-010|
+|Bottom panel                      |Aluminum |• 15": 6-39-NP5S3-012<br/>• 17": 6-39-PD7S3-012|
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/addw3/external-overview.md]: 4dc6a8f

**Review by leviport — APPROVED** [review]
:rocket:

**Pull request merged**